wallet/src/de/schildbach/wallet/util/AddressUtil.java

```diff
@@ -0,0 +1,25 @@
+package de.schildbach.wallet.util;
+
+import org.bitcoinj.core.Address;
+import org.bitcoinj.core.AddressFormatException;
+import org.bitcoinj.core.NetworkParameters;
+import org.bitcoinj.params.TestNet3Params;
+
+import de.schildbach.wallet.Constants;
+
+public class AddressUtil {
+
+    public static NetworkParameters getParametersFromAddress(String address) throws AddressFormatException {
+        NetworkParameters networkParameters = Address.getParametersFromAddress(address);
+        if (networkParameters.equals(TestNet3Params.get())) {
+            return Constants.NETWORK_PARAMETERS;
+        } else {
+            return networkParameters;
+        }
+    }
+
+    public static Address fromBase58(NetworkParameters params, String base58) throws AddressFormatException {
+        NetworkParameters networkParameters = (params != null) ? params : Address.getParametersFromAddress(base58);
+        return Address.fromBase58(networkParameters, base58);
+    }
```

Comment:
This method appears to do the same thing as Address.toBase58 from dashj.

Did this change solve any issues?

Reply:
Oh snap! Sorry for that silly mistake! There should be _getParametersFromAddress_ instead of _getParametersFromAddress_.

Fixed.